medium/src/posts/Author.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import '../styles/author.css';
+import PropTypes from 'prop-types';
+
+
+export default class Author extends Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    minutesToRead: PropTypes.number,
+    image: PropTypes.string.isRequired,
+}
+render() {
```

Comment:
More of a nitpick, but try to indent functions so that they're easier to read next time. It makes it easier for code reviews 👍